Entity/CustomItem.php

```diff
@@ -169,6 +170,11 @@ class CustomItem extends FormEntity implements UniqueEntityInterface
      */
     private $customItemLowerReferences;
 
+    /**
+     * @var string|null
+     */
+    private $uniqueHash;
```

Comment:
```suggestion
    private ?string $uniqueHash = null;
```
We can use typed properties nowadays.